app/utils/database.py

```diff
@@ -462,6 +531,30 @@ def is_admin(self):
         """
         return 'admin' in [p['level'] for p in self.privileges]
 
+    @property
+    def is_manager(self):
```

Comment:
Can we get a little reasoning (in PR is fine) about the naming convention here; why is it is_manager (which implies booleanity to me) and not genebanks_managed (or preferably a better name)?

Reply:
yeah... it's named that way since it used to be a boolean ^^ Then I changed it to give extra info. I should separate the two though!